README.md

```diff
@@ -0,0 +1,34 @@
+# ftrack web widget
+
+This small library encapsulates some logic which can be used to build custom dashboard widgets for ftrack.
+
+## Installation
+
+We recommend installing and bundling with NPM:
+
+```javascript
+npm install @ftrack/web-widget
+```
+
+## Usage
+
+To use the library, define two functions `onWidgetLoad` and `onWidgetUpdate`
+and initialize the library once the document is ready:
+
+```javascript
+import ftrackWidget from "@ftrack/web-widget";
+
+/** Initialize widget once DOM has loaded. */
+window.addEventListener("DOMContentLoaded", function onDomContentLoaded() {
+  ftrackWidget.initialize({
+    onWidgetLoad: onWidgetLoad,
```

Comment:
Out of scope, but feels like this module is missing some basic API reference. When is `onWidgetLoad` and `onWidgetUpdate` called, and with what arguments?

Reply:
Agreed. I think for public repositories it makes sense for us to convert them to Typescript and [TSDoc](https://tsdoc.org/) as soon as possible, since it significantly helps with usage documentation, and use something like [typedoc](https://typedoc.org/) to generate API references.

Reply:
Sounds good. The default output of typedoc looks a crappy, but at least the information is there. It would also be good with something that has support for usage guides and maybe even interactive playgrounds. Mainly thinking about the javascript sdk (née api 😉) here.

Anyway, separate topic to revisit soon.

Reply:
Agreed! I'm very much hoping to find a better suited theme for tsdoc, yes, or something better :)